doc/manual/src/glossary.md

```diff
@@ -149,13 +162,13 @@
     [output path]: #gloss-output-path
 
   - [deriver]{#gloss-deriver}\
-    The deriver of an *output path* is the store
-    derivation that built it.
+    The [store derivation] that produced an [output path].
 
   - [validity]{#gloss-validity}\
-    A store path is considered *valid* if it exists in the file system,
-    is listed in the Nix database as being valid, and if all paths in
-    its closure are also valid.
+    A [store path] is *valid* if all of the following hold:
+    - The store path leads to an existing [store object] in that [store].
+    - The store path is listed in the Nix database as being valid.
+    - All paths in the store path's [closure] are valid.
```

Comment:
This is only valid for the local store. For a remote store, validity is defined by a `.narinfo` being present. Really the definition of validity is that the implementation says a path is valid.